app/shared/components/composition-search/MusicSearch.test.tsx

```diff
@@ -42,23 +45,26 @@ describe('MusicSearch component', () => {
       genre: ['Романс', 'Мистецька пісня']
     }
   ];
+
   beforeEach(() => {
     render(<MusicSearch onFilterChange={handleChange} data={mockMusicData} />);
   });
 
   it('should render the MusicSearch icon', () => {
     expect(screen.getByAltText('search')).toBeInTheDocument();
   });
-  it('should render the Autcomplete and check list opening and closing', async () => {
+
+  it('should render the Autocomplete and check list opening and closing', async () => {
```

Comment:
```suggestion
  it('should render the Autocomplete and check list opening and closing', () => {
```
You delete `await` in 63 line so now `async` here is unused
So delete it here too, please